src/db_models/CollegetownHoursBase.py

```diff
@@ -0,0 +1,16 @@
+from sqlalchemy import Column, Integer, String
+from sqlalchemy.ext.declarative import declared_attr
+
+
+class CollegetownHoursBase(object):
+    @declared_attr
+    def __tablename__(cls):
+        class_name = cls.__name__  # e.g. PokelavaHour
+        table_name = class_name[0].lower() + class_name[1:] + "s"  # e.g. pokelavaHours
+        return table_name
```

Comment:
Could there be a base class that defines this `tablename` function that `CampusHoursBase` and `CollegetownHoursBase` both extend from?